src/components/stream/Stream.php

```diff
@@ -19,6 +19,8 @@
  * Class Stream
  *
  * @package Lucille\Components\Stream
+ *
+ * phpcs:disable PSR1.Methods.CamelCapsMethodName
```

Comment:
This class declares methods in `snake_case`, and we can not rename them without breaking backwards-compatibility. That is why I disable the `PSR1.Methods.CamelCapsMethodName` sniff here.